cmd/golangBuild.go

```diff
@@ -259,7 +259,10 @@ func runGolangBuild(config *golangBuildOptions, telemetryData *telemetry.CustomD
 
 		utils.SetOptions(repoClientOptions)
 
+		var binary_artifacts piperenv.Artifacts
```

Comment:
Nit: Isn't camel case the more Go ideomatic way to have multi word identifier?

Reply:
Yes. Will change it